bin/shpec

```diff
@@ -1,41 +1,51 @@
-#!/usr/bin/env bash
-VERSION=0.1.2
-examples=0
-failures=0
-test_indent=0
-red="\033[0;31m"
-green="\033[0;32m"
-norm="\033[0m"
+#
```

Comment:
A shell script traditionally started with just ': '.  Before kernels interpreted the hasbang,
the shell would see if the first character(s) equalled ':' or ': '.  In that case it would fork itself and feed itself the script.

I don't know how I ended up adding an empty comment up there.